plugins/woocommerce-admin/client/products/product-page.scss

```diff
@@ -26,8 +26,8 @@
 	.interface-interface-skeleton__header,
 	.interface-interface-skeleton__footer {
 		background-color: $white;
-		width: calc(100% - $admin-menu-width);
-		left: $admin-menu-width;
+		width: calc(100% - $admin-menu-width-collapsed);
+		left: $admin-menu-width-collapsed;
```

Comment:
Instead of setting this to the default `width` and `left` could we make use of the `folded` class that gets added to the body tag when the menu is collapsed?
You might be able to do something within here like:
```
&.folded .interface-interface-skeleton__header, &.folded .interface-interface-skeleton__footer  {
    width: calc(100% - $admin-menu-width-collapsed);
    left: $admin-menu-width-collapsed;
}
```

Reply:
Good call! I changed that in the commit [0937df1](https://github.com/woocommerce/woocommerce/pull/38941/commits/0937df11757f96a15b72333a1a94de5e54b5f4e7)